doc/source/ray-overview/index.rst

```diff
@@ -0,0 +1,160 @@
+.. _gentle-intro:
+
+============================
+A Gentle Introduction to Ray
+============================
+
+.. include:: basics.rst
+
+This tutorial will provide a whirlwind tour of all that Ray has to offer.
+
+First, install Ray with: ``pip install ray``, now we can execute some Python in parallel.
+
+Parallelizing Python Functions with Ray Tasks
+=============================================
+
+First, import ray and ``init`` the Ray service.
+Then decorate your function with ``@ray.remote`` to declare that you want to run this function 
+remotely. Lastly, call that function with the ``.remote`` method call, this yields futures that you can then 
+fetch with ``ray.get``.
+
+.. code-block:: python
+
+    import ray
+    ray.init()
+
+    @ray.remote
+    def f(x):
+        return x * x
+
+    futures = [f.remote(i) for i in range(4)]
+    print(ray.get(futures)) # [0, 1, 4, 9]
+
+In the above code block we defined some Ray Tasks. While these are great for stateless operations sometimes you
+must maintain the state of your application. You can do that with Ray Actors.
+
+Parallelizing Python Functions with Ray Actors
+==============================================
+
+Ray provides Actors to allow you to parallelize an instance of a class in Python.
+When you instantiate a class that is a Ray Actor, Ray will automatically distribute
+that actor onto the Ray cluster and manage that state on your behalf. All you as a 
```

Comment:
Hmm, Ray doesn't manage the state on your behalf, you still have to write the code to do it